CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set (oboe_sources
     src/flowgraph/ClipToRange.cpp
     src/flowgraph/ManyToMultiConverter.cpp
     src/flowgraph/MonoToMultiConverter.cpp
+    src/flowgraph/MultiToMonoConverter.cpp
```

Comment:
Couldn't both `MultiToMono` and `MonoToMulti` be contained within single `ChannelCountConverter` which takes an arbitrary number of input and output channels?

Reply:
I added a ChannelCountConverter for the previous unsupported case, eg "2:3". But it looks a bit more complex and is slower than the Mono nodes. So I kept them.